packages/govuk-react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@slice-and-dice/govuk-react",
-  "version": "0.7.4",
```

Comment:
There's a few more steps required here:
- For each of the individual components that have been updated to use the new colours, you'll need to 'yarn build' and publish to slice-and-dice
- You also need to update any references to the packages you publish. E.g. top-nav references button, so after publishing button, you need to update the top-nav reference and build and publish that too.
- Finally, update all the references in this main govuk-react/package.json to use the slice-and-dice packages.
This is so that when we import this main package in FE, it will use all the slice-and-dice components with updated colours and not still be referencing the govuk-react components listed here.

Reply:
great, thanks!